src/routes/SafeAppLandingPage/components/CreateNewSafe.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react'
+import { Link } from 'react-router-dom'
+import { Button, Text } from '@gnosis.pm/safe-react-components'
+import styled from 'styled-components'
+
+import SuccessSvg from 'src/assets/icons/safe-created.svg'
+import { OPEN_SAFE_ROUTE, SAFE_ROUTES } from 'src/routes/routes'
+
+type CreateNewSafeProps = {
+  safeAppUrl: string
+}
+
+const CreateNewSafe = ({ safeAppUrl }: CreateNewSafeProps): ReactElement => {
```

Comment:
This component is displayed when no compatible safes are available (from service and localstorage):
<img width="430" alt="Captura de pantalla 2022-05-04 a las 14 50 01" src="https://user-images.githubusercontent.com/26763673/166684565-037dd306-c991-4c2b-a021-31cab67c48d4.png">